oci/container.go

```diff
@@ -184,3 +184,7 @@ func (c *Container) State() *ContainerState {
 	defer c.opLock.Unlock()
 	return c.state
 }
+
+func (c *Container) partialIDMatch(id string) bool {
```

Comment:
We have a data structure for this. Take a look at how we look up containers in getContainerFromRequest.